src/dataset-builder/DatasetBuilder.test.ts

```diff
@@ -3,7 +3,7 @@ import userEvent from '@testing-library/user-event';
 import _ from 'lodash/fp';
 import { h } from 'react-hyperscript-helpers';
 import { DataRepo, DataRepoContract, DatasetModel } from 'src/libs/ajax/DataRepo';
-import { Cohort, ConceptSet, DatasetBuilder, DatasetBuilderContract } from 'src/libs/ajax/DatasetBuilder';
+import { Cohort, ConceptSet } from 'src/libs/ajax/DatasetBuilder';
```

Comment:
Can we join this line and line 5?

Reply:
I'm not sure what you mean

Reply:
Oh, wait, no, I am wrong! Nevermind, disregard.